src/ui/components/shared/IconWithTooltip.module.css

```diff
@@ -32,3 +32,8 @@
 .icon-tooltip {
   font-size: var(--font-size-large);
 }
+
+.iconWithTooltipButton div {
```

Comment:
Lot of styles are now unused in this file.

The view that imports this references:
* ~~iconTooltip~~ (this style doesn't actually exist)
* ~~iconWithTooltip~~ (this style doesn't actually exist)
* iconWithTooltipButton

That means these styles aren't referenced anymore and should be deleted from this CSS file:
* icon-with-tooltip
* icon-with-tooltip-button
* icon-with-tooltip

(Basically everything except the width/height)